sdk/eventhub/azure-eventhub/migration_guide.md

```diff
@@ -31,13 +34,16 @@ The code samples in this migration guide use async APIs.
 | `EventHubClientAsync.from_connection_string()` | `EventHubProducerClient.from_connection_string` or `EventHubConsumerClient.from_connection_string` |[client creation](./samples/async_samples/client_creation_async.py) |
 | `EventProcessorHost()`| `EventHubConsumerClient(..., checkpoint_store)`| [receive events using checkpoint store](./samples/async_samples/recv_with_checkpoint_store_async.py) |
 
+In V5, the SDK provides `BlobCheckpointStore`
```

Comment:
BlobCheckPointStore isn't in v5, it has been removed from v5 into a separate package

Reply:
Changed to "In V5, the SDK provides `BlobCheckpointStore` in extension packages azure-eventhub-checkpointstoreblob (for sync) and azure-eventhub-checkpointstoreblob-aio (for async).
You can define your own `CheckpointStore` class to persist checkpoint data."